en/lessons/specifics/ets.md

```diff
@@ -1,5 +1,5 @@
 ---
-version: 1.0.2
+version: 1.0.3
 title: Erlang Term Storage (ETS)
 redirect_from:
   - /lessons/specifics/ets/
```

Comment:
I would remove this redirect from new lessons. All it is doing is causing trouble when people translate lessons and it adds no benefit at all.

Reply:
Good catch @eksperimental 👍

Reply:
My bad, this is an old lesson. For some reason I thought this was a brand new lesson. So the directive is needed.

Reply:
@doomspork But it will be valid for the new Bypass lesson, https://github.com/elixirschool/elixirschool/commits/master/en/lessons/libraries/bypass.md